package.json

```diff
@@ -72,7 +72,6 @@
         "express-winston": "^4.0.2",
         "express-ws": "^4.0.0",
         "formik": "^2.2.5",
-        "fsevents": "^1.2.11",
```

Comment:
I'm not sure why this was removed, but `npm install --upgrade amqplib` is what did this.